superset/reports/models.py

```diff
@@ -147,25 +147,27 @@ class ReportSchedule(Model, AuditMixinNullable):
     # (Alerts/Reports) Unlock a possible stalled working state
     working_timeout = Column(Integer, default=60 * 60 * 1)
 
-    # Store the selected dashboard tabs etc.
-    extra = Column(Text, default="{}")
```

Comment:
Oh. I see that the `ExtraJSONMixin` is the one which defines the column as `extra_json`.